libr/core/cmd_open.c

```diff
@@ -483,6 +505,30 @@ static void map_list(RIO *io, int mode, RPrint *print, int fd) {
 	}
 }
 
+static bool _map_add_flags (void *user, void *data, ut32 id) {
+	int *flags = (int *)user;
+	RIOMap *map = (RIOMap *)data;
+
```

Comment:
asuming map and flags will never be null?

Reply:
yep, it's ensured that those are never null